frame/staking/src/mock.rs

```diff
@@ -148,25 +227,39 @@ impl pallet_balances::Trait for Test {
 	type OnFreeBalanceZero = Staking;
 	type OnReapAccount = System;
 	type OnNewAccount = ();
-	type Event = ();
+	type Event = MetaEvent;
 	type TransferPayment = ();
 	type DustRemoval = ();
 	type ExistentialDeposit = ExistentialDeposit;
 	type TransferFee = TransferFee;
 	type CreationFee = CreationFee;
 }
 parameter_types! {
-	pub const Period: BlockNumber = 1;
 	pub const Offset: BlockNumber = 0;
 	pub const UncleGenerations: u64 = 0;
 	pub const DisabledValidatorsThreshold: Perbill = Perbill::from_percent(25);
 }
+
+pub struct Babe;
+impl sp_runtime::BoundToRuntimeAppPublic for Babe {
+	type Public = dummy_sr25519::AuthorityId;
+}
+
+// insert one dummy key and one key as a representative of the babe key which we will use in reality
+// in here.
+sp_runtime::impl_opaque_keys! {
+	pub struct SessionKeys {
+		// pub foo: UintAuthorityId,
```

Comment:
Ideally I should bring this back, even though it seems like it is not really breaking anything.